p2p/chain.py

```diff
@@ -705,8 +666,8 @@ def _test() -> None:
 
     async def exit_on_sigint():
         await sigint_received.wait()
-        await syncer.cancel()
         await peer_pool.cancel()
+        await syncer.cancel()
```

Comment:
Why is this order important? Seems worth a comment.

Reply:
This is just the script to manually test the syncer, and I don't remember why I did that. In practice the syncer will usually be shut down before the `PeerPool` -- that's what happens in `FullNodeSyncer`. IOW, it doesn't really matter